src/test/java/LoginTests.java

```diff
@@ -1,20 +1,61 @@
+import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
 import java.time.Duration;
 
-public class LoginTests {
+public class LoginTests extends BaseTest{
 
-    @Test
-    public static void LoginEmptyEmailPasswordTest () {
 
-        WebDriver driver = new ChromeDriver();
+    @Test(priority = 0)
+    public void LoginEmptyEmailPasswordTest () {
+
+        Assert.assertEquals(driver.getCurrentUrl(), url);
+    }
+
+    @Test(priority = 1)
+    public void LoginValidEmailValidPasswordTest () {
+
+        provideEmail("demo@class.com");
+        providePassword("te$t$tudent");
+        clickSubmitBtn();
+
+        WebElement avatarIcon = driver.findElement(By.xpath("//img[contains(@alt,'Avatar of')]"));
+        Assert.assertTrue(avatarIcon.isDisplayed());
+
```

Comment:
I suggest putting it in a separate method example 
public boolean isDisplayedAvatar(){
WebElement avatarIcon = driver.findElement(By.xpath("//img[contains(@alt,'Avatar of')]"));
        return avatarIcon.isDisplayed();
}

Reply:
and checked via softAssertions